packages/validator/docs/actions.test.ts

```diff
@@ -1,234 +1,20 @@
+import * as data from './data/actions';
```

Comment:
I would move the `data` folder inside the tests because that is the place they should be in.
Optionally `data` rename it to `fixtures`.

The docs templates would then read the file from /test/fixtures

Reply:
I updated this in 5780f37, tests were moved to `test` directory, `data` folder was renamed to `fixtures` and moved into `test` directory. Templates for generating the docs are now in `docs/src` with extension `.md.src` as `template` is already used for validator configuration files in this package.